sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/student-loan-balances/student-loan-balances-part-time-integration.scheduler.ts

```diff
@@ -30,55 +26,35 @@ export class StudentLoanBalancesPartTimeIntegrationScheduler extends BaseSchedul
     super(schedulerQueue, queueService);
   }
 
-  /**
-   * To be removed once the method {@link process} is implemented.
-   * This method "hides" the {@link Process} decorator from the base class.
-   */
-  async processQueue(): Promise<string | string[]> {
-    throw new Error("Method not implemented.");
-  }
-
-  /**
-   * When implemented in a derived class, process the queue job.
-   * To be implemented.
-   */
-  protected async process(): Promise<string | string[]> {
-    throw new Error("Method not implemented.");
-  }
-
   /**
    * Process Student Loan Balances files from the SFTP
    * and update the database with processed records.
    * @param job Student Loan Balances job.
    * @returns processing result.
    */
-  @Process()
-  async processStudentLoanBalancesFiles(job: Job<void>): Promise<string[]> {
-    const processSummary = new ProcessSummary();
-    try {
-      processSummary.info("Processing Student Loan Balances files.");
-      const auditUser = this.systemUsersService.systemUser;
-      const serviceProcessSummary = new ProcessSummary();
-      processSummary.children(serviceProcessSummary);
-      await this.studentLoanBalancesProcessingService.processStudentLoanBalances(
-        serviceProcessSummary,
-        auditUser.id,
-      );
-      processSummary.info("Completed processing Student Loan Balances files.");
-      return getSuccessMessageWithAttentionCheck(
-        ["Process finalized with success."],
-        processSummary,
-      );
-    } catch (error: unknown) {
-      const errorMessage = "Unexpected error while executing the job.";
-      processSummary.error(errorMessage, error);
-      return [errorMessage];
-    } finally {
-      this.logger.logProcessSummary(processSummary);
-      await logProcessSummaryToJobLogger(processSummary, job);
-    }
+  protected async process(
+    _job: Job<void>,
+    processSummary: ProcessSummary,
+  ): Promise<string> {
+    processSummary.info("Processing Student Loan Balances files.");
+    const auditUser = this.systemUsersService.systemUser;
```

Comment:
May not be part of the PR. But the audit user seems to be not effective here and can be used directly in processing service.

Reply:
Yeah, not really part of the PR but I am trying to change as possible. I will be making the change.

Reply:
Refactored.

Reply:
Thanks for the refactor.